prombench/manifests/prombench/benchmark/3b_prometheus-test_deployment.yaml

```diff
@@ -66,12 +66,19 @@ spec:
         args: [
           "--web.external-url=http://{{ .DOMAIN_NAME }}/{{ .PR_NUMBER }}/prometheus-pr",
           "--storage.tsdb.path=/prometheus",
+          "--storage.tsdb.no-lockfile",
```

Comment:
Nit: Can you format it differently so future maintainers can more easily understand  e.g.
```
# default prombench args
          "--web.external-url=http://{{ .DOMAIN_NAME }}/{{ .PR_NUMBER }}/prometheus-release",
          "--storage.tsdb.path=/prometheus",
          "--web.console.libraries=/etc/prometheus/console_libraries",
          "--web.console.templates=/etc/prometheus/consoles",
          "--config.file=/etc/prometheus/prometheus.yml",
        # JSON log format is needed for GKE to display log levels correctly.
        "--log.format=json",
        "--log.level=debug",
# following args are copied from https://github.com/GoogleCloudPlatform/prometheus-engine/blob/722ee81b87348557d7274be0dccfa14c3987b9af/cmd/operator/deploy/operator/10-collector.yaml#L92
# ensure both are in sync, exclude flags that are not relevant.
        - --enable-feature=exemplar-storage
        - --storage.tsdb.no-lockfile
        # Keep 30 minutes of data. As we are backed by an emptyDir volume, this will count towards
        # the containers memory usage. We could lower it further if this becomes problematic, but
        # it the window for local data is quite convenient for debugging.
        - --storage.tsdb.retention.time=30m
        - --storage.tsdb.wal-compression
        # Effectively disable compaction and make blocks short enough so that our retention window
        # can be kept in practice.
        - --storage.tsdb.min-block-duration=10m
        - --storage.tsdb.max-block-duration=10m
        - --export.user-agent-mode=kubectl
```